apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/shared/use-draft-entity-subgraph.ts

```diff
@@ -18,7 +18,7 @@ export const useDraftEntitySubgraph = (
   Dispatch<SetStateAction<Subgraph<EntityRootType> | undefined>>,
   boolean,
 ] => {
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
drive-by: without this, you could get a flash of error state when viewing the `create-entity-page', since both `loading` and `draftEntitySubgraph` start off false.